cleanup/mavenSnapshotCleanupWhenRelease/README.md

```diff
@@ -0,0 +1,59 @@
+Artifactory Maven Snapshot Cleanup When Release
+=====================================
+
+This plugin deletes Maven SNAPSHOT artifacts when the release version is published.
+
+
+Features
+--------
+
+Consider the Maven snapshots repository `maven-local-lib-snapshots` populated like that:
+
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-1.pom
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-1.jar
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-2.pom
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-2.jar
+
+    org/jfrog/test/my-artifact/1.1.0-SNAPSHOT/my-artifact-1.1.0-20170202.080829-1.pom
+    org/jfrog/test/my-artifact/1.1.0-SNAPSHOT/my-artifact-1.1.0-20170202.080829-1.jar
+
+
+When the **pom file** of `my-artifact` version `1.0.0` is deployed into `maven-local-lib-releases` (Maven releases repository), the associated snapshot is deleted.
+
+e.g: folder `org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/` of `maven-local-lib-snapshots` repository.
+
+Installation
+------------
+
+To install this plugin:
+
+* Place files `mavenSnapshotCleanupWhenRelease.groovy` and `mavenSnapshotCleanupWhenRelease.properties` under the master Artifactory server `${ARTIFACTORY_HOME}/etc/plugins`.
+
+* Configure the file `mavenSnapshotCleanupWhenRelease.properties` with the repositories couples *releases <-> snapshots*. In a couple, the first should be the *release* repository and the second the *snapshot* repository:
+    
+    repositories =  [ ["maven-local-lib-releases","maven-local-lib-snapshots"], ["maven-local-plugin-releases","maven-local-plugin-snapshots"] ]
+ 
+* Configure the logger in `${ARTIFACTORY_HOME}/etc/logback.xml` with level wanted:
+
+    <configuration ...>
+        ...
+        <logger name="mavenSnapshotCleanupWhenRelease">
+            <level value="info"/>
+        </logger>   
+    </configuration>
```

Comment:
These snippets don't appear to show correctly when the markdown is rendered. I know indenting four spaces denotes a code block, but I've always had trouble getting Github to recognize such. Instead, try surrounding your snippets in [triple backticks](https://github.github.com/gfm/#fenced-code-blocks).